traefik-hub/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik-hub
 description: Traefik Hub Ingress Controller
 type: application
-version: 2.2.1
+version: 2.3.0
```

```suggestion
version: 2.2.1
```
I suggest to avoid publish a new version on each PR.